tests/CI/elasticsearch:6.6.0.env

```diff
@@ -0,0 +1,8 @@
+discovery.type=single-node
```

Comment:
These 3 env files look awfully similar to me. The only difference is `plugins.security.disabled=true` (why, btw ? ) and `OPENSEARCH_JAVA_OPTS`. IT doesn't work to just add these two options in every case ? It certainly would be clearer and more readable to keep the environment in the docker compose file

Reply:
These environments are destined to be more and more different one from the others, as ElasticSearch and OpenSearch are quickly drifting apart, so think the chosen solution is a good one.

`plugins.security.disabled=true` is only there for running the tests, as by default OS imposes you SSL and username+password. In production it's obviously different.